bridge-sdk/connectors/eth-connector/src/eth_connector.rs

```diff
@@ -49,6 +49,7 @@ pub struct EthConnector {
 
 impl EthConnector {
     /// Transfers ETH to the EthCustodian and sets recipient as a Near account. A proof from this transaction is then used to mint nETH on Near
+    #[tracing::instrument(skip_all, name = "deposit to near")]
```

Comment:
"deposit to near" -> "DEPOSIT TO NEAR"

It is like a header, I think easier to read, if it is in caps. The same comment for all other functions.

Reply:
Done